cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ApplicationsOperations.java

```diff
@@ -112,6 +109,31 @@ public ApplicationsOperations(DefaultCloudFoundryOperations cloudFoundryOperatio
             .onErrorStop();
     }
 
+    /**
+     * Prepares a request for pushing an app to the cloud foundry instance specified
+     * within the cloud foundry operations instance. The resulting mono is
+     * preconfigured such that it will perform logging.
+     * Essentially this method is equivalent to the create method.
```

Comment:
It should be explained in which way it differs from the `create` method (it first removes the currently deployed application and newly pushes it afterwards)